src/Processors/Formats/Impl/PrettyBlockOutputFormat.cpp

```diff
@@ -397,14 +432,24 @@ static String highlightDigitGroups(String source)
 
 void PrettyBlockOutputFormat::writeValueWithPadding(
```

Comment:
- The name of function `writeValueWithPadding` does not represent the purpose of this function really good because, as far as I understood, here we write only the first row of values, decides if we have a break line and also perform calculations for the next rows. So maybe we can find more reasonable function name.
- The good solution can also be to divide this function into some parts, because the next function `writeTransferredRow` contains the same parts of this function.

Reply:
The fact is that both of these functions write data, and the purpose of `writeTransferredRow` is fully reflected in its name. At the same time, `writeValueWithPadding` not only writes data, but also decides whether we will use breakLine, and this is a bit unobvious at the first sight. I think we can skip this issue for now because it's hard to find the most proper way to fix it and it is not that critical.🙂

Reply:
I think I found the solution, now there is no decision logic in `writeValueWithPadding` when find a break line. Avoided code duplication in `writeValueWithPadding` and `writeTransferredRow`